sdcm/logcollector.py

```diff
@@ -1136,26 +1136,30 @@ def get_aws_instances_by_testid(self):
                                                    instance=instance,
                                                    global_ip=instance['PublicIpAddress'],
                                                    tags={**self.tags, "NodeType": "monitor", }))
-        # Temporarily disabling cloud-manager logs collection
-        # due to: https://github.com/scylladb/scylla-cluster-tests/issues/3816
-        if self.params["use_cloud_manager"] and False:  # pylint: disable=condition-evals-to-constant
+        if self.params["use_cloud_manager"]:
             try:
-                from cluster_cloud import get_manager_instance_by_cluster_id  # pylint: disable=import-outside-toplevel
+                from cluster_cloud import get_aws_manager_instance_by_cluster_id  # pylint: disable=import-outside-toplevel
```

Comment:
i'm not sure i follow this change. what happens if it is a GCP siren cluster. does it also try importing this AWS method?